### PR TITLE
Clean up layouts

### PR DESCRIPTION
**Pull request by burhan-syed**
Cleans up layouts and zindex issues with chat & profile header

### CODE REVIEW

File: src/components/Profile/ProfileHeader.tsx [hunk 1]
```diff
@@ -31,78 +27,72 @@ export function ProfileHeader({ profileId }: { profileId: string }) {
...
+              user.data?.bannerPhotoUrl
+                ? user.data?.bannerPhotoUrl
+                : "https://cdn.thewirecutter.com/wp-content/media/2021/06/20210617_doggie_dna_topart_2x1.jpg?auto=webp&quality=75&crop=1.91:1&width=1200"
+            }
+            alt="Background"
+            width={700}
+            height={200}
+            className="h-full max-h-52 max-w-full object-cover"
+            unoptimized={true}
+          ></Image>
```
Comment (redconOne) [inline src/components/Profile/ProfileHeader.tsx]: Isn't Image supposed to be self-closing?

**Review by malloryporche — DISMISSED** [review]
Looks great!

File: src/components/Chat/AddChat.tsx [hunk 2]
```diff
@@ -6,32 +6,34 @@ export default function AddChat({ userId }: { userId?: string }) {
...
+        className="btn btn-circle"
+        disabled={mutation.isLoading}
+        aria-label="chat"
+        onClick={() =>
+          userId &&
+          mutation.mutate(
+            { userId },
+            {
+              onError(error, variables, context) {
+                if (error.message === "UNAUTHORIZED") {
```
Comment (squeakypickles33) [inline src/components/Chat/AddChat.tsx:20]: line 20 and 24 variables and context necessary?

File: src/components/Chat/AddChat.tsx [hunk 3]
```diff
@@ -6,32 +6,34 @@ export default function AddChat({ userId }: { userId?: string }) {
...
-      onClick={() =>
-        userId &&
-        mutation.mutate(
-          { userId },
-          {
-            onError(error, variables, context) {
-              if (error.message === "UNAUTHORIZED") {
-                setDisplayLoginModal(true);
-              }
+    <div className="tooltip" data-tip="message">
```
Comment (squeakypickles33) [inline src/components/Chat/AddChat.tsx:9]: Semantics but the tip could be more clear "Send Message"

File: src/pages/index.tsx [hunk 4]
```diff
@@ -1,24 +1,16 @@
...
 
 export default function Home() {
-  const hello = api.example.hello.useQuery({ text: "example hi" });
-
   return (
     <>
       <Head>
-        <title>MyApp</title>
-        <meta name="description" content="App description" />
+        <title>PetPals</title>
```
Comment (squeakypickles33) [inline src/pages/index.tsx:8]: this is on me but are we PetPals or PetCentral?

**Review by squeakypickles33 — APPROVED** [review]

**Pull request merged**